### PR TITLE
Fix Wigner small d-matrix

### PR DESCRIPTION
**Pull request by gnrraphi**
**Brief description of what is fixed or changed**

The current maximum and minimum values of sigma 
https://github.com/sympy/sympy/blob/69f98fb2b0d845e76874067a381dba37b577e8c5/sympy/physics/wigner.py#L1087-L1088
in the `wigner_d_small` function are incorrect. This discrepancy becomes evident when examining the arguments of the binomial coefficients, as shown in
https://github.com/sympy/sympy/blob/69f98fb2b0d845e76874067a381dba37b577e8c5/sympy/physics/wigner.py#L1093-L1094
A prime illustration of this issue is observed when evaluating the expression `wigner_d_small(2, 0)`.  This example highlights an error arising due to an erroneous inclusion of a term that leads to division by zero. In this pull request, I fixed the values of sigma, respectively .

**Release notes**

* physics.wigner
  * Fixed a bug with the `wigner_d_small` function.

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -1519,6 +1519,7 @@ erdOne <36414270+erdOne@users.noreply.github.com>
 extraymond <extraymond@gmail.com>
 faizan2700 <syedfaizan824@gmail.com>
 foice <foice.news@gmail.com>
+gnrraphi <raphael.lehner@gmail.com>
```
Comment (oscarbenjamin) [inline .mailmap:1522]: Is this the name that you want to have recorded in the AUTHORS file?

You could add this as
```
Raphael Lehner <raphael.lehner@gmail.com> gnrraphi <raphael.lehner@gmail.com>
```
Then the .mailmap file will show Raphael Lehner but the mailmap check scrpt will understand that commits recorded as `gnrraphi <raphael.lehner@gmail.com>` belong to that same person.

**Comment by oscarbenjamin** [comment]
I don't know anything about Wigner matrices but I trust that you do.

One or two regression tests should be added for whatever is fixed by this. Perhaps here:
https://github.com/sympy/sympy/blob/4604615322eff993bf0aebb6c197d794128506f6/sympy/physics/tests/test_clebsch_gordan.py#L58

File: sympy/physics/wigner.py [hunk 2]
```diff
@@ -1084,8 +1084,8 @@ def wigner_d_small(J, beta):
         for j, Mj in enumerate(M):
 
             # We get the maximum and minimum value of sigma.
-            sigmamax = max([-Mi-Mj, J-Mj])
-            sigmamin = min([0, J-Mi])
+            sigmamax = min([J-Mi, J-Mj])
```
Comment (faze-geek) [inline sympy/physics/wigner.py:1087]: What is the motivation behind this change? Can you point me to some failing examples ... or maybe resources showing this as the expected value?

Comment (gnrraphi) [reply sympy/physics/wigner.py:1087]: The formula for the Wigner small d-matrix can be found in section 4.3.1, equation (2) of [https://doi.org/10.1142/9789814415491_0005](url) or by examining the implemented code. In this formula, the sum should only include values where all factorials are non-negative, meaning that the binomial coefficients are not equal to zero. However, in the current code, this condition is not satisfied, leading to terms of the form 0/0, as seen in the case of 
 `J=1`, `Mi=1` , `Mj=-1` , and `beta=0`. This results in division by `sin(0)`, which equals 0.

**Comment by oscarbenjamin** [comment]
Looks good to me. Thanks!

**Pull request merged**